website_and_docs/content/documentation/webdriver/interactions/print_page.en.md

```diff
@@ -177,4 +181,42 @@ Using `getBackground()` and `setBackground()` methods, you can get/set whether t
 {{< tab header="Kotlin" >}}
 {{< badge-implementation >}}
 {{< /tab >}}
-{{< /tabpane >}}
\ No newline at end of file
+{{< /tabpane >}}
+
+## Printing
+
+Once you've configured your PrintOptions, you're ready to print the page. To do this, 
+you can invoke the print function, which generates a PDF representation of the web page. 
+The resulting PDF can be saved to your local storage for further use or distribution. 
+Using `PrintsPage()`, the print command will return the PDF data in base64-encoded format, which can be decoded 
+and written to a file in your desired location, and using `BrowsingContext()` will return a String. 
+
+There may currently be multiple implementations depending on your language of choice. For example, with Java you
+have the ability to print using either `BrowingContext()` or `PrintsPage()`. Both take `PrintOptions()` objects as a
+parameter.
+
+Note: `BrowsingContext()` is part of Selenium's BiDi implementation. To enable BiDi see [Enabling Bidi]({{< ref "bidi/" >}}) 
+
+{{< tabpane text=true >}}
+{{% tab header="Java" %}}
+**PrintsPage()**
+{{< gh-codeblock path="examples/java/src/test/java/dev/selenium/interactions/PrintsPageTest.java#L13-L20" >}}
```

Comment:
I believe the code lines have been modified. Could you please make the necessary adjustments @shbenzer?

Reply:
Yep, done.